addons/wowl/static/src/web_enterprise/webclient.ts

```diff
@@ -0,0 +1,34 @@
+import { WebClient } from "../webclient/webclient";
+import { EnterpriseNavBar } from "./navbar/navbar";
+import { makeHomeMenuService, useHomeMenuToggle } from "./navbar/home_menu";
+import { serviceRegistry } from "../services/service_registry";
+
+if (document.location.pathname.includes("/wowlent")) {
+  odoo.edition = "enterprise";
+  serviceRegistry.add("home_menu", makeHomeMenuService());
```

Comment:
we'll also need a file that export the webclient Class to mount (a file that would be community/enterprise specific), s.t. there is no magic in main.ts